packages/plugin-core/src/_extension.ts

```diff
@@ -379,7 +364,24 @@ export async function _activate(
         return p;
       }
     );
-    const { port } = await startServer();
+    const { port, subprocess } = await startServerProcess();
+    if (subprocess) {
+      WSUtils.handleServerProcess({
+        subprocess,
+        context,
+        onExit: () => {
+          const txt = "Restart Dendron";
+          vscode.window
+            .showErrorMessage("Dendron engine encountered an error", txt)
+            .then(async (resp) => {
+              if (resp === txt) {
+                AnalyticsUtils.track(VSCodeEvents.ServerCrashed);
```

Comment:
any short error message context that we can add to the analytics event? 

Longer term I'll investigate the backend error reporting strategy